cry.bat

```diff
@@ -24,19 +24,19 @@ CALL :build "%~dp0"
 GOTO :eof
 
 :clean
-CALL python3 %~f1\Scripts\bet.py --clean
+CALL python %~f1\Scripts\bet.py --clean
 GOTO :eof
 
 :build
-CALL python3 %~f1\Scripts\bet.py --build
+CALL python %~f1\Scripts\bet.py --build
 GOTO :eof
 
 :make
 CALL msbuild sad.sln -property:Configuration=Debug
 GOTO :eof
 
 :tests
-CALL python3 %~f1\Scripts\bet.py --tests
+CALL python %~f1\Scripts\bet.py --tests
```

Comment:
This is for group discussion - I don't want to disrupt workflows unnecessarily.

My computer has `python3` on the path and I don't know how many people aliased their `python` to `python3` in lieu of this 😂

Down with this if everyone is changing this already on their local machine, but if everyone is setup with `python3` it might be nice to just rename this locally and avoid committing it.